#
# Upstream: http://code.google.com/p/chromium/issues/detail?id=3805
# Ubuntu: none
# Description: on dual core computers, linking the two test_shell* binaries
# at the same time consumes nearly 2GB of memory. Use only 1 cpu/core.
#
---
 build/SConscript.main |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

Index: chrome/build/SConscript.main
===================================================================
--- chrome.orig/build/SConscript.main
+++ chrome/build/SConscript.main
@@ -119,19 +119,17 @@
 root_env.AddMethod(ChromeObject)
 
 
 # TODO(bradnelson): pull this functionality into hammer.
 # Auto select the number of processors
 if root_env['PLATFORM'] in ['win32', 'cygwin']:
   cpus = int(os.environ.get('NUMBER_OF_PROCESSORS', 1))
 elif root_env['PLATFORM'] in ['linux', 'linux2', 'posix']:
-  # TODO(evanm): this is Linux-specific, not posix.
-  # Parse /proc/cpuinfo for processor count.
-  cpus = len([l for l in open('/proc/cpuinfo') if l.startswith('processor\t')])
+  cpus = 0
 else:
   cpus = 1
 SetOption('num_jobs', cpus + 1)
 
 
 if ARGUMENTS.get('VERBOSE') in (None, '0'):
   root_env['CCCOMSTR'] = 'Compiling $TARGET ...'
   root_env['CXXCOMSTR'] = 'Compiling $TARGET ...'
